explainers/navigation-to-unsigned-bundles.md

```diff
@@ -0,0 +1,363 @@
+# Explainer: Navigation to Unsigned Web Bundles<br>(a.k.a. Bundled HTTP Exchanges)
+
+Last updated: Feb 04, 2020
+
+Participate at https://github.com/WICG/webpackage and
+https://datatracker.ietf.org/wg/wpack/about/.
+
+Users want to be able to share content with their friends, even when neither has
+an internet connection. Some websites, for example some books, encyclopedias,
+and games, want to help their users share them in this way. For individual
+images or videos, users have native apps like
+[SHAREit](https://www.ushareit.com/), [Xender](http://www.xender.com/), or
+[Google Files](https://files.google.com/) that can share files.
+
+<!-- TOC -->
+
+- [Proposal](#proposal)
+  - [Relevant structure of a bundle](#relevant-structure-of-a-bundle)
+  - [Process of loading a bundle](#process-of-loading-a-bundle)
+  - [Loading a trusted unsigned bundle.](#loading-a-trusted-unsigned-bundle)
+  - [Loading an untrusted bundle](#loading-an-untrusted-bundle)
+  - [Subresource loads with an attached bundle](#subresource-loads-with-an-attached-bundle)
+  - [Service Workers](#service-workers)
+  - [URLs for bundle components](#urls-for-bundle-components)
+- [Open design questions](#open-design-questions)
+  - [Network access](#network-access)
+- [Security and privacy considerations](#security-and-privacy-considerations)
+- [Considered alternatives](#considered-alternatives)
+  - [Alternate formats considered](#alternate-formats-considered)
+    - [Save as directory tree](#save-as-directory-tree)
+    - [Save as MHTML](#save-as-mhtml)
+    - [Save as Web Archive](#save-as-web-archive)
+    - [WARC](#warc)
+    - [Mozilla Archive Format](#mozilla-archive-format)
+    - [A bespoke ZIP format](#a-bespoke-zip-format)
+  - [Alternate URL schemes considered](#alternate-url-schemes-considered)
+    - [arcp](#arcp)
+    - [pack](#pack)
+- [Stakeholder feedback](#stakeholder-feedback)
+- [Acknowledgements](#acknowledgements)
+
+<!-- /TOC -->
+
+## Proposal
+
+We propose to use the [Web Bundle
+format](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html),
+without signatures, to give users a format they can create from any website
```

Comment:
I'm slightly worried if this may sound like it excludes any types of signatures, e.g. raw-key signed bundles or app-level signatures... while some use cases would need that.  (While, it may be good to start with this limited focus)